src/components/TodoForm.js

```diff
@@ -37,8 +37,8 @@ function TodoForm(props) {
                         value={input}
                         name="text"
                         className="todo-input edit"
-                        onChange={handleChange}
-                        ref={inputRef}
```

Comment:
removal of `ref={inputref}` caused reference error, now inputRef.current.focus(); won't work anymore